.github/workflows/build.yml

```
@@ -11,10 +11,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.3)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)